.gitlab-ci.yml

```diff
@@ -0,0 +1,212 @@
+stages:
+- runall
+- build
+- collate
+
+# This is just a config to help trigger rest of the builds
+run_all_builds:
+  image: ubuntu:latest
+  stage: runall
+  variables:
+    GIT_STRATEGY: none
+  script:
+  - pwd
+  when: manual
+  allow_failure: false
+
+# Build
+build_osx_runtime:
+  stage: build
+  tags:
+    - buildfarm
+    - darwin
+  script:
+  - git submodule update --init --recursive
+  - echo -e "[auth]\nono.schemes=https\nono.prefix=ono.unity3d.com\nono.username=$GITLAB_UNITY_META_ONO_USER\nono.password=$GITLAB_UNITY_META_ONO_PASS\n" >> ~/.hgrc
```

Comment:
I think we only needed this for mono-build-deps clone from Ono. Can maybe be removed?